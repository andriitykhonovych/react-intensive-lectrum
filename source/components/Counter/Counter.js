// Core
import React from "react";
import PropTypes from "prop-types";

// Instruments
import styles from "./styles.m.css";

const Counter = ({ count }) => {
    return <section className = { styles.counter }>Post count: {count}</section>;
};

Counter.ptopTypes = {
    posts: PropTypes.string,
};

export default Counter;
