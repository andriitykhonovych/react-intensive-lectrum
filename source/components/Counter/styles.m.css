.counter {
    color: var(--paletteColor7);
    display: flex;
    justify-content: center;
    margin-bottom: 10px;
    position: relative;
}