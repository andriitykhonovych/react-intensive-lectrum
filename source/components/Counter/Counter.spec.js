// Core
import React from 'react';
import dom from 'react-test-renderer';

// Components
import Counter from './Counter';

const tree = dom.create(<Counter count = { 3 } />).toJSON();

test('Counter component should correspond to their snapshot', () => {
    expect(tree).toMatchSnapshot();
});
