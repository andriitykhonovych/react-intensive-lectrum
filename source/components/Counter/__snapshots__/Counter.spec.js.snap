// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter component should correspond to their snapshot 1`] = `
<section
  className="counter"
>
  Post count: 
  3
</section>
`;
